plugins/babel/src/run-babel.ts

```diff
@@ -0,0 +1,44 @@
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { hookConsole } from '@dotcom-tool-kit/logger'
+import type { BabelOptions } from '@dotcom-tool-kit/types/src/schema/babel'
+import * as babel from '@babel/core'
+import fg from 'fast-glob'
+import { promises as fs } from 'fs'
+import path from 'path'
+import type { Logger } from 'winston'
+
+export async function runBabel(
+  logger: Logger,
+  options: BabelOptions,
+  transformOptions?: babel.TransformOptions
+): Promise<void> {
+  const fileGlob = options.files ?? 'src/**/*.js'
+  const files = await fg(fileGlob)
+  // Work out the root of the glob so we can strip this part of the path out of
+  // the outputted files.
+  // E.g., a glob of 'src/**/*.js'   = src/a/b.js -> lib/a/b.js
```

Comment:
**praise**: helpful comment 👍 

**suggestion**: think it'll be useful to log out that we're running babel with the options here, so when people do npm run start and have babel configured into their run:local hook, they can see that babel is being run in their terminal